src/data/disks.rs

```diff
@@ -7,9 +7,9 @@ use std::path::{Path, PathBuf};
 /// the names of the disks are generated dynamically.
 #[derive(Default, Debug, Clone, PartialEq, Eq)]
 pub struct Disk {
-    mount_point: PathBuf,
-    display_name: String,
-    is_removable: bool,
+    pub mount_point: PathBuf,
+    pub display_name: String,
+    pub is_removable: bool,
```

Comment:
Is there anything wrong with leaving it private and creating access methods? It should not be intended for such an object to be edited again after it has been created.

Reply:
Added a constructor, made these private again.